mypy.ini

```diff
@@ -93,6 +93,3 @@ ignore_missing_imports = True
 
 [mypy-flask_sqlalchemy.*]
 ignore_missing_imports = True
-
-[mypy-awsgi.*]
-ignore_missing_imports = True
```

Comment:
We might have to add the same for apig-wsgi eventually. Not blocking this review.